portal/token-authentication/README.md

```diff
@@ -1,25 +1,26 @@
 # Token authentication
 
-Access a map service that is secured with an ArcGIS token-based authentication.
+Access a web map that is secured with ArcGIS token-based authentication.
```

Comment:
This has changed to say "access a web map", but the 'use case' specifies a 'service'. Should we keep service because it is more broad?